py-rse/packaging.Rmd

```diff
@@ -75,23 +76,27 @@ pkg_name
 └── setup.py
 ```
 
-which in our case would mean changing the name of `bin/` to `zipf`.
+The top-level directory is named after the package.
+It contains a directory that is also named after the package,
+and that contains the package's source files.
+It is initially a little confusing to have two directories with the same name,
+but most Python projects follow this convention because
+it makes it easier to set up the project for installation.
+We can get this structure in our Zipf's Law project
+simply by renaming `zipf/bin` to `zipf`.
```

Comment:
```suggestion
by renaming `zipf/bin` to `zipf`.
```

Reply:
Also, should we actually write out the command?
```
$ mv bin/ zipf/
```